nautobot/ipam/migrations/0051_vlangroup_tags.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 4.2.15 on 2024-09-26 15:54
+
+from django.db import migrations
+
+import nautobot.core.models.fields
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("extras", "0118_remove_task_queues_from_job_and_queue_from_scheduled_job"),
+        ("ipam", "0050_vlangroup_range"),
```

Comment:
Would it make sense to merge this migration into `0050_vlangroup_range` rather than having two consecutive VLANGroup schema migrations?

Reply:
I merged them 👌